README.md

```diff
@@ -8,6 +8,16 @@
 - [How To Build](https://wiki.status.im/Building_status-go)
 - [Notes on Bindings](https://wiki.status.im/Status-go_Binding_notes)
 
+# Notes on Git/GitHub workflow
+
+1. Squash PR before merging.
+   You can do it either with GitHub api or locally if you want to preserve signature.
```

Comment:
We can do be more explicit with `with Github API by merging using "Squash and merge" or locally [...]`.

Also `preserve your signature`.